ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -7,6 +7,7 @@
 @Builder
 public class CCDCase {
 
+    private Long id;
```

Comment:
is there are reason for this to not be a string?

Reply:
any reason to make this String?

Reply:
swagger spec for ccd says it's a string.

Reply:
actually, that's fine. nevermind.